view/src/components/foodbanks.js

```diff
@@ -0,0 +1,937 @@
+            aria-labelledby="customized-dialog-title"
+            open={viewOpen}
+            fullWidth
+            classes={{ paperFullWidth: classes.dialogStyle }}
+          >
+            <DialogTitle id="customized-dialog-title" onClose={handleViewClose}>
+              {this.state.foodbankName}
+            </DialogTitle>
+            <DialogContent dividers>
+              <Chip
+                className={classes.chip}
+                label="High Food Insecurity"
+                size="small"
+              />
+              <Chip className={classes.chip} label="Major City" size="small" />
+              <Box
+                display="flex"
+                flexDirection="row"
+                flexWrap="wrap"
+                p={0}
+                m={0}
+              >
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Details:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Location of Food Bank:
+                    {`${this.state.location.substring(0, 30)}`}
+                    <br />
+                    {`${this.state.location.substring(30, 78)}`}
+                    Regrigeration Space (in pallets):
+                    {this.state.refrigerationSpaceAvailable}
+                    <br />
+                    Max Load Size (in pallets): {this.state.maxLoadSize}
+                  </Typography>
+                </Box>
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Point of Contact:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Name: {this.state.contactName}
+                    <br />
+                    Phone: {this.state.contactPhone}
+                    <br />
+                    Email: {this.state.contactEmail}
+                  </Typography>
+                </Box>
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Logistics:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Forklift: {this.state.forklift ? "yes" : "no"}
+                    <br />
+                    Pallet: {this.state.pallet ? "yes" : "no"}
+                    <br />
+                    Loading Dock: {this.state.loadingDock ? "yes" : "no"}
+                  </Typography>
+                </Box>
+                <Box p={3}>
+                  <Typography className={classes.pos} color="textSecondary">
+                    Hours of Operation:
+                  </Typography>
+                  <Typography variant="body2" component="p">
+                    Monday-Friday: 9am - 5pm
+                    <br />
+                    Saturday: 10am - 4pm
+                    <br />
+                    Sunday: closed
```

Comment:
Suspicious...  I think I'd recommend hiding this section or leaving it blank until you're ready to implement it, rather than putting bogus data in there.  Juuuuuust in case we forget about it before the end of the internship.  :)

Reply:
hahaha great point! Is it ok if I comment it out for now?